adapters/v1/armo.go

```diff
@@ -0,0 +1,74 @@
+package v1
+
+import (
+	"fmt"
+	"net/http"
+	"strconv"
+
+	sysreport "github.com/armosec/logger-go/system-reports/datastructures"
+	pkgcautils "github.com/armosec/utils-k8s-go/armometadata"
+	"github.com/kubescape/kubevuln/core/domain"
+	"github.com/kubescape/kubevuln/core/ports"
+)
+
+type ArmoAdapter struct {
+	clusterConfig pkgcautils.ClusterConfig
+}
+
+var _ ports.Platform = (*ArmoAdapter)(nil)
+
+func NewArmoAdapter(accountID, eventReceiverURL string) *ArmoAdapter {
+	return &ArmoAdapter{
+		clusterConfig: pkgcautils.ClusterConfig{
+			AccountID:            accountID,
+			EventReceiverRestURL: eventReceiverURL,
+		},
+	}
+}
+
+const ActionName = "vuln scan"
+const ReporterName = "ca-vuln-scan"
+
+var details = []string{
+	sysreport.JobStarted,
+	sysreport.JobStarted,
+	sysreport.JobSuccess,
+	sysreport.JobDone,
+}
+var statuses = []string{
```

Comment:
@Daniel-GrunbergerCA I did an indirection to abstract from implementation

Reply:
I see. I think you should check for that the index exists in the list in the `SendStatus` function just in case, so we don't panic. Also I think is better if you move the lists to the steps.go file, since it will be easier to understand how these lists are used and what is the purpose of the steps consts.

Reply:
no, the list is implementation specific, it should not spill out